.github/workflows/build-test-webeditor.yml

```diff
@@ -76,11 +76,10 @@ jobs:
         working-directory: github1s
       - run: yarn serve > ./output 2>&1 &
         working-directory: github1s
-      - name: Set up JDK 11
-        uses: actions/setup-java@v3
+      - uses: actions/setup-java@v3
         with:
           distribution: 'zulu'
-          java-version: 11
+          java-version: 17
```

Comment:
Why 17? I suppose there would be some issues with 17. Any compelling reason to upgrade to 17?

Reply:
Nope